src/styles/utils/variables.scss

```diff
@@ -0,0 +1,10 @@
+$quantity-boxes: 6;
+
+$colors: (
+  rgb(255, 0, 0): 1,
+  rgb(204, 0, 0): 2,
+  rgb(153, 0, 0): 3,
+  rgb(102, 0, 0): 4,
+  rgb(51, 0, 0): 5,
+  rgb(0, 0, 0): 6,
+);
```

Comment:
you can try to calculate the red part of RGB in scss loop, where you iterate indexes.